### DISCUSSION TITLE
Where to start here Junior Java?

### DISCUSSION THREAD

**Discussion by DinaDurykina** [community/community#22591] [New to GitHub]
<p>Hi! I’m a Junior. just mastered java and kotlin. I really want to write code that someone needs. Please, tell me where to start? Thanks</p>

**Comment by mpboom** [comment, marked as answer]
<p>Nice of you to think about contributing to open source! The topics at GitHub are a great place to start looking for projects:</p>
<p><a href="https://github.com/topics/" rel="nofollow noopener">https://github.com/topics/</a></p>
<p>You can for example, browse Java projects at:</p>
<p><a href="https://github.com/topics/java" rel="nofollow noopener">https://github.com/topics/java</a></p>
<p>or, find some Kotlin-specific projects at:</p>
<p><a href="https://github.com/topics/kotlin" rel="nofollow noopener">https://github.com/topics/kotlin</a></p>
<p>Good luck! Just open a PR if you think you’ve built something nice - you’ll learn from it even if it doesn’t get merged.</p>
